### PR TITLE
docs: add new argoproj maintainers

### PR DESCRIPTION
**Pull request by alexmt**
Signed-off-by: Alexander Matyushentsev <AMatyushentsev@gmail.com>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -34,4 +34,12 @@
 | Hong Wang | [wanghong230](https://github.com/wanghong230) | Reviewer | [Akuity](https://akuity.io/) | Yes |
 | Jonathan West| [jgwest](https://github.com/jgwest) | Approver - CD | [Red Hat](https://www.github.com/redhat/) | Yes |
 | May Zhang | [mayzhang2000](https://github.com/mayzhang2000) | Approver - CD | [Intuit](https://www.github.com/intuit/) | Yes |
-| Andrii Perenesenko | [perenesenko](https://github.com/perenesenko) | Reviewer - Rollouts | [Intuit](https://www.github.com/intuit/) | Yes |
\ No newline at end of file
+| Andrii Perenesenko | [perenesenko](https://github.com/perenesenko) | Reviewer - Rollouts | [Intuit](https://www.github.com/intuit/) | Yes |
+| Daniel Soifer | [daniel-codefresh](https://github.com/daniel-codefresh) | Reviewer - Events Argo Events | [Codefresh](https://www.github.com/codefresh/) | Yes |
+| Hari Rongali | [harikrongali](https://github.com/harikrongali) | Reviewer - Events | [Intuit](https://www.github.com/intuit/) | Yes |
```
Comment (hblixt) [inline MAINTAINERS.md:39]: This should be Rollouts.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -34,4 +34,12 @@
 | Hong Wang | [wanghong230](https://github.com/wanghong230) | Reviewer | [Akuity](https://akuity.io/) | Yes |
 | Jonathan West| [jgwest](https://github.com/jgwest) | Approver - CD | [Red Hat](https://www.github.com/redhat/) | Yes |
 | May Zhang | [mayzhang2000](https://github.com/mayzhang2000) | Approver - CD | [Intuit](https://www.github.com/intuit/) | Yes |
-| Andrii Perenesenko | [perenesenko](https://github.com/perenesenko) | Reviewer - Rollouts | [Intuit](https://www.github.com/intuit/) | Yes |
\ No newline at end of file
+| Andrii Perenesenko | [perenesenko](https://github.com/perenesenko) | Reviewer - Rollouts | [Intuit](https://www.github.com/intuit/) | Yes |
+| Daniel Soifer | [daniel-codefresh](https://github.com/daniel-codefresh) | Reviewer - Events Argo Events | [Codefresh](https://www.github.com/codefresh/) | Yes |
```
Comment (hblixt) [inline MAINTAINERS.md:38]: Typo  - Should be just "Events"

**Review by hblixt — CHANGES_REQUESTED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (alexmt) [reply MAINTAINERS.md]: thank you! fixed

File: MAINTAINERS.md (hunk 2 above)
Comment (alexmt) [reply MAINTAINERS.md]: fixed

**Review by edlee2121 — APPROVED** [review]

**Review by hblixt — APPROVED** [review]

**Pull request merged**